.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -23,7 +23,7 @@ Please do not remove items from the checklist
 - [ ] Update the release branch:
   - [ ] Update `RELEASE_BRANCH` and `RELEASE_VERSION` in `Makefile` and run `make prepare-release-branch`
   - [ ] Update the `CHANGELOG`
-  - [ ] Submit a pull request with the changes: <!-- example #211 #214 -->
+  - [ ] Submit a pull request with the changes: <!-- PREPARE_PULL --> <!-- example #211 #214 -->
```

Comment:
@mimowo Could you please update this line manually in release issues: https://github.com/kubernetes-sigs/kueue/issues/6413, https://github.com/kubernetes-sigs/kueue/issues/6414

Reply:
Done, thanks. Don't resolve the comment for future visibility. cc @tenzen-y

Reply:
Thank you!